### ISSUE TITLE
Cursorless Scopes Sidebar Not Working for a Specific File

### ISSUE THREAD

**Issue by AndrewDant** [cursorless-dev/cursorless#1996]
I have an individual talon file (various_tasks.talon) for which the scope sidebar does not seem to be working at all, despite working for my other talon files. The full text of that file is in the block below. If I switch to the tab with that file from another file I will continue to see the scopes from the prior file, and if I reload the window with that tab open I will see no scopes at all. After switching from another tab if I click on a scope that I know exists in the file I get an 'No match for pattern' error message in the bottom right.

```
app: vscode
-
generate docs: key(alt-j)

quick fix: key(ctrl-.)

copy command id: user.copy_command_id()

pipeline: user.vscode_with_plugin("gl.pipelineActions")
open [active] file in browser: user.vscode_with_plugin("gl.openActiveFile")

search next: user.vscode("search.action.focusNextSearchResult")
search last: user.vscode("search.action.focusPreviousSearchResult")
search file remove: user.vscode("search.searchEditor.action.deleteFileResults")
search remove: user.vscode("search.action.remove")

cursorless record: user.vscode("cursorless.recordTestCase")

open recent: user.vscode("workbench.action.openRecent")

change language:
    user.vscode_with_plugin("workbench.action.editor.changeLanguageMode")
    
toggle word wrap:
    user.vscode_with_plugin("editor.action.toggleWordWrap")
    
(show file|open file): user.vscode_with_plugin("extension.openFileFromPath")

jest: user.vscode_with_plugin("editor.action.triggerSuggest")

toggle hats: user.vscode("cursorless.toggleDecorations")

toggle blame: user.vscode("gitlens.toggleFileBlame")

revert [selected] range: user.vscode("git.revertSelectedRanges")
```

**Comment by pokey** [comment]
works fine for me. maybe another CRLF issue?

**Comment by pokey** [comment]
yeah breaks when I switch on CRLF. prob tree-sitter-talon crlf issue cc/ @wenkokke re https://github.com/wenkokke/tree-sitter-talon/pull/42

**Comment by pokey** [comment]
We should also solve the meta-problem here which is better error tolerance for our scope view

**Comment by rokubop** [comment]
may also be related to https://github.com/cursorless-dev/cursorless/issues/1776

**Comment by pokey** [comment]
Fixed in VSCode parse tree extension version 0.28.1. Filed https://github.com/cursorless-dev/cursorless/issues/2004 to track general error tolerance of scope sidebar
